src/main.rs

```diff
@@ -675,8 +675,8 @@ fn run_watch(opt: &Opt, config: &Config, interval: u64) -> Result<(), Error> {
         let _ = tx.send(());
     });
 
+    let mut term_info = term_info();
     'outer: loop {
-        let term_info = term_info();
```

Comment:
I think this line is required because term size may be changed by each loop.

Reply:
That's a good point. I'll try resizing my terminal to see what happens there.

Reply:
Yeah, this patch breaks that behavior. Let me fix this...